app/presenters/publication_presenter.rb

```diff
@@ -48,10 +48,15 @@ def updated_at
     DateTime.parse(date) if date
   end
 
-  def to_json
-    {
-      places: places
-    }.to_json
+  def video_embed_url
```

Comment:
Sorry at first I thought you were referring to the `to_json` method, now I realise it's about the `video_embed_url` method.  This has been removed in another PR, which is why I think this looks like it's being added.  Now removed it again 👍